R-package/R/lgb.cv.R

```diff
@@ -83,6 +85,7 @@ lgb.cv <- function(params = list()
                    , showsd = TRUE
                    , stratified = TRUE
                    , folds = NULL
+                   , train_folds = NULL
```

Comment:
Can you please move this down to after `reset_data`? Adding new arguments in the middle of the signature is a breaking change for anyone using positional arguments.

Reply:
Yes, you are right. Didn't think about that.

Reply:
However, should I keep the docs in the current order (this is, right after `folds`?) This way we don't break compatibility but feels more natural for users

Reply:
Indeed, it does fail. So the new argument should go at the end in both the docs and the function signature.

Reply:
yep, thanks for investigating!